mappings/net/minecraft/block/material/Material.mapping

```diff
@@ -0,0 +1,61 @@
+CLASS none/ayo net/minecraft/block/material/Material
+	CLASS none/ayo$1
+		METHOD c isSolid ()Z
+	FIELD A CACTUS Lnone/ayo;
+	FIELD B OTHER Lnone/ayo;
+	FIELD C PLANT_BLOCK Lnone/ayo;
```

Comment:
Essentially, `PLANT_BLOCK` is meant for full blocks: Melon and pumpkin. I addressed this in the notes, and I agree.